src/main/java/uk/gov/hmcts/et/common/model/multiples/MultipleData.java

```diff
@@ -2,11 +2,16 @@
 
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.annotation.JsonProperty;
+
+import lombok.AllArgsConstructor;
```

Comment:
Think you have a checkstyle issue here
```suggestion
import lombok.AllArgsConstructor;
```